GPflow/likelihoods.py

```diff
@@ -436,7 +436,7 @@ def predict_mean_and_var(self, Fmu, Fvar):
             # To compute this, we'll compute the density for each possible output
             possible_outputs = [tf.fill(tf.stack([tf.shape(Fmu)[0], 1]), np.array(i, dtype=np.int64)) for i in
                                 range(self.num_classes)]
-            ps = [self.predict_density(Fmu, Fvar, po) for po in possible_outputs]
+            ps = [tf.exp(self.predict_density(Fmu, Fvar, po)) for po in possible_outputs]
```

Comment:
so as the predict density class is now creating correct log probabilities I turn them back into normal probabilities here: do you think this is okay or is there a more stable way to do this?